### ISSUE TITLE
NTR: Hyperoxia test abnormal (Cardiac & vascular)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7789]
New term request Hyperoxia test abnormal (brumme_220419172139): Oxygenation saturation does not improve on 100% FiO2 compared to 21% FiO2, Hyperoxia failure, hyperoxemia test, abnormal oxygen saturation, sweating with feeds can indicate a need for test, includes congenital cardiac disease  UMLS: 2262749

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000228
- label: Failure to increase oxygen saturation on hyperoxia test
- parents: [Abnormal cardiovascular system physiology (HP:0011025)](https://hpo.jax.org/app/browse/term/HP:0011025)
- pmid: PMID:29763177
- def: Hyperoxia test is the initial method to distinguish critical congenital heart disease (CCHD) from pulmonary disease. The test consists in measuring an arterial blood gas at room air and 100% inspired oxygen after 10 minutes. Neonates with congenital heart disease are usually not able to increase PaO2 above 100 mm Hg during 100% oxygen administration. In patients with pulmonary disease, PaO2 generally increased greater than or equal to 100 mm Hg with 100% oxygen as ventilation-perfusion discrepancies are overcome. A positive result indicates the cardiac origin and further cardiac workup is indicated to rule out CCHD.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7789
- synonyms:
